src/main.c

```diff
@@ -28,7 +28,10 @@ char *brd_decode_str(char *src, char *dst)
         if (!src[i])
             break;
         char n = ~(4*src[i] | (src[i] & 0xC0) >> 6);
+            
         if (n==(char)0xD7)
+            n = '\0';
+        if(n == (char)0xCB)
```

Comment:
Could you please preserve the original formatting? Specifically, not add an empty line 31, add one space after `if` and remove spaces around `==`. Thanks!